### DISCUSSION TITLE
how to get vertical slider to fit its inner content height

### DISCUSSION THREAD

**Discussion by ElijahWoelbing** [nolimits4web/swiper#5352] [Q&A]
I am trying to get the vertical swiper to fit its content, it works as expected in horizontal mode.
heres a sandbox I made, if you change the mode from horizontal to vertical you will see what I mean
https://codesandbox.io/s/swiper-vertical-forked-q5glk?file=/index.html

**Comment by rtabulov** [comment]
i think `slidesPerView: 'auto'` should work for you

https://codesandbox.io/s/swiper-vertical-forked-i8wbr?file=/index.html
